packages/trade-widget/src/App.css

```diff
@@ -0,0 +1,28 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 80px;
+}
+
+.App-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.App-title {
+  font-size: 1.5em;
+}
+
+.App-intro {
+  font-size: large;
+}
+
+@keyframes App-logo-spin {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
```

Comment:
I'm really not a fan of CSS. It's too easy to shoot yourself in the foot and have designs that are brittle because changing the CSS styling of a class changes the styling in many disparate places in the app. Can we continue the convention of in-lining the styling within the React components where used?